src/style.css

```diff
@@ -1 +1,40 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+div.stars--1 > *:first-child {
```

Comment:
Avoid using tag names for styling. Instead of using 'div.stars--1', consider using just '.stars--1'. This aligns with the checklist item: 'Don't use tag names for styling (except html and body)'.